infra/gcp/terraform/k8s-infra-oci-proxy-prod/oci-proxy.tf

```diff
@@ -99,7 +99,7 @@ resource "google_cloud_run_service" "oci-proxy" {
 
         // ensure this macth the value for template.spec.containers.resources.limits
         env {
-          name = "GOMAXPROCS"
+          name  = "GOMAXPROCS"
```

Comment:
unrelated space change?

Reply:
This is a result of running a lint command with the Terraform binary: `terraform fmt .`.